tests/tests_suite_test.go

```diff
@@ -124,6 +125,12 @@ func (s *newSspStrategy) Init() {
 			CommonTemplates: sspv1beta1.CommonTemplates{
 				Namespace: s.GetTemplatesNamespace(),
 			},
+			TektonPipelines: &sspv1beta1.TektonPipelines{
+				Namespace: s.GetNamespace(),
```

Comment:
no, the regex is there only because of one serviceAccount. Tekton does not deploy SA with name pipeline in openshift|kube namespaces. This SA is needed for running pipelines from UI. So pipelines can be deployed to any namespaces, but only when deployed to openshift|kube namespace, operator has to dpeloy extra SA.

Reply:
Ok, thanks for the explanation.
@codingben , can you use a namespace that passes the regex, so that functests can check that the extra SA is deployed correctly?

Reply:
Yes, very nice @akrejcir that you spotted it. Thanks.